dapp/src/components/ProjectInfo.astro

```diff
@@ -0,0 +1,188 @@
+---
+import Topic from "./utils/Topic.astro";
+import Modal from "./utils/Modal.astro";
+---
+
+<div class="relative flex flex-col items-center md:flex-row justify-between">
+  <Topic title="Project Information" description="" id="project-name-topic"/>
```

Comment:
Sorry maybe I was not clear. This should look like this.
```suggestion
  <Topic title="Project" description="Details about the project" id="project-name-topic"/>
```
And when you get the name it should appear in another div right above the list of maintainers. So around L25 here.

Can be like you have in the modal to update the config basically.